packages/testcontainers/src/container-runtime/clients/container/docker-container-client.ts

```diff
@@ -158,6 +158,7 @@ export class DockerContainerClient implements ContainerClient {
     try {
       log.debug(`Fetching container logs...`, { containerId: container.id });
       const stream = (await container.logs({
+        ...opts,
         follow: true,
```

Comment:
But, if we are doing this now

```typescript
const stream = (await container.logs({
        ...opts,
        follow: true,
        stdout: true,
        stderr: true,
        since: opts?.since ?? 0,
})) as IncomingMessage;
```

at the end, like we are applying  this property setting: after destructuring opts:

````typescript
 follow: true,
 stdout: true,
 stderr: true,
 since: opts?.since ?? 0
 ````
 
 we can pass for example the follow as false and he will be set always to true so, probably you want implement this in a way that allow passing any option with any value the user wants?
 
 You can review the snippet https://jsbin.com/gofedomanu/edit?js,console to take a look the behavior that I'm talking about.

Reply:
reverted to previous version